packages/app/src/cli/commands/app/dev.ts

```diff
@@ -92,6 +94,7 @@ export default class Dev extends Command {
     await addPublicMetadata(() => ({
       cmd_app_dependency_installation_skipped: flags['skip-dependencies-installation'],
       cmd_app_reset_used: flags.reset,
+      cmd_dev_tunnel_type: flags['tunnel-url'] ? 'custom' : flags.tunnel,
```

Comment:
the tunnel type was only being reported after a successful dev execution, but I think is worth it to report it BEFORE, just in case the dev crashes. That way we can know if it crashes for a specific tunnel type